### PR TITLE
[WIP] MMUT

### PR DESCRIPTION
**Pull request by Biswajitghosh98**
Implemented Leapfrog Magnus as mentioned in the issue https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/issues/1062

### CODE REVIEW

File: src/OrdinaryDiffEq.jl [hunk 1]
```diff
@@ -198,7 +198,7 @@ module OrdinaryDiffEq
          Kvaerno3, KenCarp3, Cash4, Hairer4, Hairer42, SSPSDIRK2, Kvaerno4,
          Kvaerno5, KenCarp4, KenCarp5, ESDIRK54I8L2SA
 
-  export MagnusMidpoint, LinearExponential
+  export MagnusMidpoint, LinearExponential, MMUT
```
Comment (ChrisRackauckas) [inline src/OrdinaryDiffEq.jl]: Let's call it MagnusLeapfrog: much more common.

Comment (Biswajitghosh98) [reply src/OrdinaryDiffEq.jl]: @ChrisRackauckas Done !

**Comment by ChrisRackauckas** [comment]
Failure unrelated

**Comment by ChrisRackauckas** [comment]
Please add docs

**Comment by Biswajitghosh98** [comment]
Fixes #1062

**Pull request merged**